src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -237,7 +237,10 @@ DlgPreferences::~DlgPreferences() {
     }
 
     // When DlgPrefControllers is deleted it manually deletes the controller tree items,
-    // which makes QTreeWidgetItem trigger this signal.
+    // which makes QTreeWidgetItem trigger this signal. If we don't disconnect,
+    // &DlgPreferences::changePage iterates on the PreferencesPage instances in m_allPages,
+    // but the pDlg objects of the controller items are already destructed by DlgPrefControllers,
```

Comment:
Hmm, I've never heard the word "destructed" used that way. I'd say "destroyed", but I don't think it really matters :man_shrugging:

Reply:
ooops, better to write correctly ;) 
Done